maquetado/tech.scss

```diff
@@ -0,0 +1,48 @@
+.react-icon {
+  height: 100px;
+  margin: 10px;
+  width: 100px;
+}
+
+.title {
+  font-size: $text-medium;
+  margin: 10px;
+}
+
+.title-relase {
+  font-size: $text-medium;
+  font-weight: bold;
+  margin: 10px;
+}
+
+.title-description {
+  font-size: $text-medium;
+  margin: 10px;
+}
+// cuadro verde
+.react-tech {
+  align-items: center;
+  background-color: $green;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  margin: 10px 4px;
+  position: relative;
+  right: -10px; // puede no ir, lo puse para que este separado del borde de la pagina
+  text-align: center;
+  top: 10px; // puede no ir, lo puse para que este separado del borde de la pagina
+  width: 200px;
+}
+// cuadro transparente
+.react-tech::before {
+  border: 3px solid $green;
+  border-radius: 10px;
+  content: '';
+  height: 300px;
```

Comment:
Change this for `100%` because if the height of the element changes, you also need to change it